front-end/src/views/login/index.vue

```diff
@@ -127,6 +133,12 @@ export default {
         this.passwordType = 'password'
       }
     },
+    handleMessage(event) {
+      const data = event.data
+      if (data.hasOwnProperty('name') && data.hasOwnProperty('accessToken')) {
+        console.log(data)
```

Comment:
Is this debug?
We are printing access tokens in the console.
Please drop this log